packages/fhir/build-dts.ts

```diff
@@ -0,0 +1,3 @@
+export default async () => {
+  console.log(' >> WOOP WOOP <<<');
```

Comment:
TODO: this my hook to use dts bundle instead of tsc for this one adaptor

Or I suppose I could consider using dts bundle for all adaptors and creating everything in one file? It'll make generation of type listings downstream a lot easier. I remember there was a discussion on this 2 years ago and non-bundling won for some reason